### DISCUSSION TITLE
Is it possible to create a site with a defined language?

### DISCUSSION THREAD

**Discussion by joselu1sc** [pnp/pnpcore#711] [Q&A] [answered]
Hi,

I have been reading and exploring some code in PnP. There are some options like  Microsoft.SharePoint.Client.CreateSiteAsync or create it with Graph API.

Well, in graph is not possible to set the language during the creation. Is there any method or example about site creations with defined languages in PnP Core documentation.

Thank you <3

**Comment by jansenbe** [comment, marked as answer]
@joselu1sc : This should be possible with the PnP Core SDK Admin library, see https://pnp.github.io/pnpcore/using-the-sdk/admin-sharepoint-sites.html#creating-site-collections for the docs. You can set the `Language` property when creating the site collection.

**Reply by joselu1sc** [reply]
Thank you! I was wondering about the reason to use Delegated permissions instead of App-Only... Thank you so much :)!
